_includes/cockroachcloud/sql-connection-string-free.md

```diff
@@ -2,7 +2,7 @@
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    cockroach sql --url 'postgresql://<username>:<password>@<serverless-host>:26257/defaultdb?sslmode=verify-full&sslrootcert='$HOME'/.postgresql/root.crt&options=--cluster=<routing-id>'
+    cockroach sql --url 'postgresql://<username>:<password>@<serverless-host>:26257/defaultdb?sslmode=verify-full&sslrootcert='$HOME'/.postgresql/root.crt'
```

Comment:
Nit: we use <> in the javascript examples but all the other examples seem to be using {} for placeholder text.

Reply:
Yeah, that's annoying. Since this is included in multiple topics that use `<>` in the non-included text, I'll have to punt on this for now.